### DISCUSSION TITLE
Contribution in  Sunbird Project

### DISCUSSION THREAD

**Discussion by FossGenTechologies** [Sunbird-Lern/Community#99] [Enhancement]
@pramodkvarma @surendrasinghs
We at Fossgen Technologies have Robotic Process Automation (RPA) team that can automate existing manual monitoring and save effort and time of IT Ops team. Please let us know how can we contribute in Sunbird Project.

**Comment by rhwarrier** [comment]
Thank you for reaching out, team.   @SanthoshVasabhaktula  could we take a look at what the options here are, and how it can help the SB efforts?
